ansible/vars.yml

```diff
@@ -10,11 +10,12 @@ postgresql_release_checksum: sha256:ea2cf059a85882654b989acd07edc121833164a30340
 pgbouncer_release: "1.19.0"
 pgbouncer_release_checksum: sha256:af0b05e97d0e1fd9ad45fe00ea6d2a934c63075f67f7e2ccef2ca59e3d8ce682
 
-# to get these use `wget https://github.com/PostgREST/postgrest/releases/download/v12.0.2/postgrest-v12.0.2-ubuntu-aarch64.tar.xz -q -O- | sha1sum`
-# note: update temporarily hardcoded value over at docker/all-in-one/Dockerfile on the next PostgREST version
-postgrest_release: "12.0.2-listeners-alpha"
-postgrest_arm_release_checksum: sha1:a61633a4118eaefd5351ad236744fd84fbb7886e
-postgrest_x86_release_checksum: sha1:d57eecdf732b7fd2a38889b35f1ed484a719d003
+# to get these use
+# wget https://github.com/PostgREST/postgrest/releases/download/v12.2.0/postgrest-v12.2.0-ubuntu-aarch64.tar.xz -q -O- | sha1sum
+# wget https://github.com/PostgREST/postgrest/releases/download/v12.2.0/postgrest-v12.2.0-linux-static-x64.tar.xz -q -O- | sha1sum
+postgrest_release: "12.2.0"
```

Comment:
https://github.com/supabase/postgres/commit/64260decf21c1163106a66c5f86255251934397e needs to be reverted

Reply:
Reverted in https://github.com/supabase/postgres/pull/1003/commits/6d01ddb9946e4e2a83108c19f5b1800352dfc035